kinto/core/storage/redis.py

```diff
@@ -342,11 +356,14 @@ def delete_all(self, collection_id, parent_id, filters=None,
                    modified_field=DEFAULT_MODIFIED_FIELD,
                    deleted_field=DEFAULT_DELETED_FIELD,
                    auth=None):
+
         records, count = self.get_all(collection_id, parent_id,
                                       filters=filters,
                                       id_field=id_field,
                                       modified_field=modified_field,
                                       deleted_field=deleted_field)
+        # XXX: Here we need to have the correct coolection_id and
+        # parent_id for the record id
```

Comment:
I don't think you can do it via `get_all()`, since for each deleted record you'll need to figure out which parent_id and collection_id it was stored into. We might have to rewrite `delete_all()` :/